test/supplemental_genbank_tests/genbank_upload_full_test.py

```diff
@@ -167,3 +171,125 @@ def test_simple_shock_upload(self):
         self.assertIsNotNone(result['genome_ref'])
         self.assertTrue(int(
             result['genome_info'][10]['Number of Protein Encoding Genes']) > 0)
+
+    def _run_test_fail(self, params, error_message, mass=True):
+        with pytest.raises(Exception) as got:
+            if mass:
+                self.getImpl().genbanks_to_genomes(self.getContext(), params)
+            else:
+                self.getImpl().genbank_to_genome(self.getContext(), params)
+        assert_exception_correct(got.value, ValueError(error_message))
+
+    def _get_objects_data(self, refs):
+        results = self.getDfuClient().get_objects({'object_refs': refs})
+        return [result["info"] for result in results["data"]]
+
+    def test_genbank_to_genome_invalid_workspace(self):
```

Comment:
What about a test where both are provided?

Are there any successful genbank to genome tests that provide a workspace ID?

Reply:
both refer to `workspace_id` and `workspace_name`?

Reply:
Right, so it should fail because both are in the input

Reply:
👍